### ISSUE TITLE
Upgrade to v46.1.0 and Events went away?

### ISSUE THREAD

**Issue by bryancass** [stripe/stripe-dotnet#2999]
### Describe the bug

Following the instructions on the Stripe docs Webhook page, using .NET, [https://docs.stripe.com/connect/webhooks](https://docs.stripe.com/connect/webhooks)

I get this error after upgrading to v46.1.0:
![image](https://github.com/user-attachments/assets/d2650930-b052-419f-9179-0640566a8c4f)

### To Reproduce

See code snippet

### Expected behavior

Why doesn't Events exist in the current context?

### Code snippets

```csharp
using Stripe;
[...]
        /// <summary>
        /// POST: WebHooks/Stripe
        /// </summary>
        /// <returns>HTTP OK or BadRequest</returns>
        [HttpPost]
        public async Task<IActionResult> Stripe()
        {
            var json = await new StreamReader(HttpContext.Request.Body).ReadToEndAsync();

            try
            {
                var stripeEvent = EventUtility.ConstructEvent(
                 json,
                 Request.Headers["Stripe-Signature"],
                 _stripeSettings.WebhookSigningKey);

                if (stripeEvent.Type = Events.FileCreated)  <- The name "Events" does not exist in the current context
                {
                    // Do something
                }
[...]
```

### OS

Windows 10

### .NET version

.NET 8.0.10

### Library version

Stipe.net v46.1.0

### API version

2024-09-30.acacia(?)

**Comment by jar-stripe** [comment]
@bryancass Very sorry for the trouble here!  Stripe.Events was renamed Stripe.EventTypes because we introduced a new Stripe.Events namespace in Stripe.net v46.  If it helps, here's a guide with other breaking changes that were introduced in V46: https://github.com/stripe/stripe-dotnet/wiki/Migration-guide-for-v46.  For this issue, we've added a task on our end to update our public docs to match the latest release of the SDK.
